schemahcl/types.go

```diff
@@ -343,6 +343,24 @@ func SizeTypeAttr(required bool) *TypeAttr {
 	}
 }
 
+// ScaleTypeAttr returns a TypeAttr for a scale attribute.
+func ScaleTypeAttr() *TypeAttr {
```

Comment:
This is a driver-specific attribute (right?). Let's keep it in mssql.

Reply:
I saw it repeated in mysql, pg too

Reply:
kk, same as "HasCollate" - let's do it in a separate PR and move it to a shared place. To keep driver PRs, mssql specific.